controllers/utils/constants.go

```diff
@@ -2,6 +2,9 @@ package utils
 
 import "k8s.io/apimachinery/pkg/runtime/schema"
 
+// DefaultCGUControllerWorkerCount default number of worker threads for the CGU controller
+const DefaultCGUControllerWorkerCount = 5
```

Comment:
Building on my earlier comment, we could bundle all the CGU Controller constants here:
```suggestion
// CGUController constants
const (
	CGUCtrllWorkerCountEnv          = "TALM_CGU_CTRL_WORKER_COUNT"
	DefaultCGUControllerWorkerCount = 5
)

```

Reply:
I noticed we still reference `talo` in both backup and pre-caching related templates. Perhaps to remain consistent, we should refer to the environment variable as  `"TALO_CGU_CTRL_WORKER_COUNT"`? Either way, I don't mind - I simply wanted to highlight this here.